auth/authRouter.js

```diff
@@ -1,15 +1,15 @@
 const express = require('express');
 const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
-
+const passport = require('passport');
 const router = express.Router();
 
 const Coaches = require('../coaches/coaches-model');
-
+const passportSetup = require('../config/passport-setup');

```

Comment:
You're assigning this to a variable, but nothing is being exported from that file. May look a little nicer if you pass `passport` as an argument:

```js
require('../config/passport-setup')(passport);
```

Reply:
Tried it with your suggestion but it cause the server to crash because it is saying `requre(...) is Not a Function`.